### DISCUSSION TITLE
Micro frontend overrides styling of Shell application.

### DISCUSSION THREAD

**Discussion by rsuthar-dev** [webpack/webpack#16061] [General]
# Bug report

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  5
Node.js version: 14
Operating System: Windows
Additional tools:

Issue Detail:

I have one Micro frontend application based om Webpack Module federation.

When a micro-app is loaded in shell application , it overrides styling of shell application because of same classname.

is there any way we can isolate styling at container level and micro-apps level ?

**Comment by alexander-akait** [comment]
> If the current behavior is a bug, please provide the steps to reproduce.

Please fill out it, thank you

**Comment by ScriptedAlchemy** [comment]
This is likely because you’re using global css and not locally scoped styles or css modules. 

Anything global can cause collisions and css cascades so last style loaded wins

**Comment by azuy212** [comment]
@rsuthar-dev were you able to fix your issue, if yes can you please explain how you fixed it?
